eoscompanion/dummy_feed.py

```diff
@@ -0,0 +1,117 @@
+                                        {
+                                            'applicationId': 'com.endlessm.celebrities.en'
+                                        }
+                                    ]
+                                }
+                            ],
+                            'detail': [
+                                {
+                                    'title': 'Lionel Messi',
+                                    'synopis': '.',
+                                    'thumbnail': '/v1/content_data?applicationId=com.endlessm.celebrities.en&deviceUUID=1234&contentId=7743ae136927867094cd8c3bffa0f13d5ef63696',
+                                    'uri': '7b4978ba82ecf6b992c395956e0adad4b4734dbe'
+                                }
+                            ]
+                        },
+                        {
+                            'contentType': 'Artwork',
+                            'source': [
+                                {
+                                    'type': 'application',
+                                    'detail': [
+                                        {
+                                            'applicationId': 'com.endlessm.wiki_art.en'
+                                        }
+                                    ]
+                                }
+                            ],
+                            'detail': [
+                                {
+                                    'title': 'Madonna with Child',
+                                    'synopis': '.',
+                                    'thumbnail': '/v1/content_data?applicationId=com.endlessm.wiki_art.en&deviceUUID=1234&contentId=19db92f13325e93e4a7af18a997ed3941722c176',
+                                    'uri': '7afc3eaf5805936f60404037112f08a41d0470cc'
+                                }
+                            ]
+                        },
+                        {
+                            'contentType': 'Video',
+                            'source': [
+                                {
+                                    'type': 'application',
+                                    'detail': [
+                                        {
+                                            'applicationId': 'com.endlessm.video_kids'
+                                        }
+                                    ]
+                                }
+                            ],
+                            'detail': [
+                                {
+                                    'title': 'Nice Crystal Ball Juggler in Europe',
+                                    'thumbnail': '/v1/content_data?applicationId=com.endlessm.video_kids&deviceUUID=1234&contentId=3dfe56e4f28bacfebe9b7a800529dc9653267eff',
+                                    'uri': 'bd794303e04e6d2bcc62858e47d813483056f88c',
+                                    'duration': '121'
```

Comment:
Technically this should be a timecode like 01:30 but since it is a string I don't think the application will do anything special with it.

Reply:
Changed to use the post-processed duration.